src/pages/2021/schedule.mdx

```diff
@@ -8,9 +8,12 @@ import { SiteLayout } from "../../components/layout/SiteLayout";
     </p>
 </div>
 
-All times listed are in Pacific Standard Time (PST). Please note that this is a high-level schedule and a more detailed schedule will be made available closer to the conference.
-  
-  
+All times listed are in Pacific Standard Time (PST). Please note that this is a high-level schedule.
+
+<div className="text-2xl flex justify-center p-8">
+  <CTAButton onClick="https://events.bizzabo.com/343425/agenda">View the agenda here!</CTAButton>
+</div>
+    
```

Comment:
There are 4 extra spaces here, this will prevent the next heading from rendering properly.

This should be a blank line, no spaces or any other whitespace character.